.circleci/config.yml

```diff
@@ -16,9 +16,26 @@ jobs:
             - node_modules
           key: v1-dependencies-{{ checksum "yarn.lock" }}
       - run: yarn build
+      - save_cache:
+          paths:
+            - build
+          key: v1-build-{{ .Environment.CIRCLE_SHA1 }}
       - run: yarn test:lint
       - run: yarn test:prettier
 
+  publish_main_tag:
+    docker:
+      - image: circleci/node:12
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-build-{{ .Environment.CIRCLE_SHA1 }}
+      - run:
+          name: Set version
+          command: cat <<< $(jq --arg v 0.0.0-main-$(echo $CIRCLE_SHA1 | cut -c -7) -r '.version |= $v' package.json) > package.json
```

Comment:
Ugly but `npm version 0.0.0-main-$(echo $CIRCLE_SHA1 | cut -c -7)` prints
```
sh: 1: standard-version: not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
...
```

It looks like the docker image is missing some dependency.

Reply:
Ok, I was just stupid.

Removed `standard-version` here: 8f59f71